spec/lib/miq_expression_spec.rb

```diff
@@ -2861,15 +2861,16 @@
     end
 
     # TODO: think this should return same results as missing model?
-    it "return column info for managed-field" do
+    it "return column info for managed-field (no model)" do
       tag = "managed-location"
       col_info = described_class.get_col_info(tag)
       expect(col_info).to match(
         :data_type                      => :string,
         :excluded_by_preprocess_options => false,
+        :format_sub_type                => :string,
         :include                        => {},
         :tag                            => true,
-        :sql_support                    => true,
+        :sql_support                    => false,
```

Comment:
This is an invalid query. I stand by this change saying it is not supported by sql.